### PR TITLE
Add translators to the credits

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

**Comment by BrayanDSO** [comment]
Just asking, does Crowdin have a public link listing all Ankidroid translators?

**Comment by Arthur-Milchior** [comment]
Can anyone not an admin let me know whether, on https://crowdin.com/project/ankidroid/members , you see a thousand of contributors, or only the few people with privileges?

FYI you can see crowdin's API on https://developer.crowdin.com/api/v2/#operation/api.user.get 
and so yes, there is.

However, we have not gathered translator permissions, and I don't want to play with giving people's name without having checked whether it's alright with them.
We can send them all a message through crowdin, so we can ask them to fill a google form and ask them whether they want to be credited for their work and what name to use. However, I've currently no idea how to deal with someone who later may want to change their name. In particular, between Google history and git history (if we download the list of names on git), it seems impossible to delete their name from history. I've no idea how to be compliant with various privacy laws here, in particular GPRD, the right to be forgotten, or just ensuring we do not keep a record of people's [deadnames](https://en.wikipedia.org/wiki/Deadnaming).

**Comment by david-allison** [comment]
Consent and GDPR/CCPA compliance lies with CrowdIn/GitHub. 

I think we're overcomplicating this. A link is fine.

File: README.md [hunk 1]
```diff
@@ -129,7 +129,9 @@ Our Awesome Team of Contributors
 <a href="https://opencollective.com/ankidroid#contributors" target="_blank"><img src="https://opencollective.com/ankidroid/contributors.svg?width=890"></a>
 
 
+### [Translators](https://opencollective.com/ankidroid#contributors)
```
Comment (david-allison) [inline README.md:132]: This link is incorrect. It should point to CrowdIn.

**Review by david-allison — CHANGES_REQUESTED** [review]
link fix needed

**Comment by david-allison** [comment]
> https://crowdin.com/project/ankidroid/members

This link is a 403 if a user is not logged in. 

Let's link to the activity stream: https://crowdin.com/project/ankidroid/activity-stream. That has the added advantage of showing people the activity and how easy it is to get started.

**Comment by Arthur-Milchior** [comment]
This stream don't allow to easily see the full list of contributors; unless you press "show more" a nonsensical number of time.

I admit it's not very important. I still believe it would have been a nice gesture to have a full list of people agreeing to it in the app.

**Comment by Arthur-Milchior** [comment]
Ready for re-review

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]

**Pull request merged**